mailer.js

```diff
@@ -226,10 +236,7 @@ module.exports = function (log) {
 
   Mailer.prototype.newSyncDeviceEmail = function (message) {
     log.trace({ op: 'mailer.newSyncDeviceEmail', email: message.email, uid: message.uid })
-    var query = {
-      email: message.email
-    }
-    var link = this.initiatePasswordResetUrl + '?' + qs.stringify(query)
+    var link = this.initiatePasswordChangeUrl
 
```

Comment:
Does this new URL not take an `email` query param?  We should file a follow-up bug to add one if so, ISTM this is useful behaviour to coerce the site into the required state.